content/toc-doc.json

```diff
@@ -219,10 +219,57 @@
       {
         "slug": "/docs/tina-cloud/api-versioning/",
         "title": "API Versioning"
+      }
+    ]
+  },
+  {
+    "title": "Self-hosting Tina",
+    "items": [
+      {
+        "title": "Introduction",
+        "slug": "/docs/self-hosted/overview/"
+      },
+      {
+        "title": "Getting Started",
+        "items": [
+          {
+            "title": "Self-Hosted Starter",
+            "slug": "/docs/self-hosted/starters/nextjs-vercel/"
+          },
+          {
+            "title": "Implementing on an Existing Site",
+            "slug": "/docs/self-hosted/existing-site/"
+          }
+        ]
+      },
+      {
+        "title": "Querying Content",
+        "slug": "/docs/self-hosted/querying-data/"
```

Comment:
Looks like this link gives a 404

Reply:
It its working for me, mind trying again?